Automattic/vip
# Refactor logout

## Description

This changes `vip logout --help` from ignoring `--help` to respect it:

```
 $ vip logout --help
  Usage: vip logout [options] 
  
  Options:
    -d, --debug    Activate debug output
    -h, --help     Output the help for the (sub)command
    -v, --version  Output the version number
  
  Examples:
    - Logs out current user.
    $ vip logout

```

This among other needed for consistent docs generation.  @yolih do you have any input on the text above, this will be then pulled to the docs site. 

## Steps to Test


1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-logout.js --help`
1. Verify cookies are delicious.
